BunqSdk.Tests/Http/ResponseIdOnBadRequestTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Bunq.Sdk.Context;
+using Bunq.Sdk.Exception;
+using Bunq.Sdk.Model.Generated.Endpoint;
+using Xunit;
+
+namespace Bunq.Sdk.Tests.Http
+{
+    public class ResponseIdOnBadRequestTest : BunqSdkTestBase
+    {
+        /// <summary>
+        /// API context to use for the test API calls.
+        /// </summary>
+        private static readonly ApiContext API_CONTEXT = GetApiContext();
+        
+        /// <summary>
+        /// Invalid user id to trigger BadRequestException
+        /// </summary>
+        private const int INVALID_USER_PERSON_ID = 0;
+
+        [Fact]
+        public void TestBadRequestWithResponseId()
+        {
+            ApiException caughtException = null;
+
+            try
+            {
+                UserPerson.Get(API_CONTEXT, INVALID_USER_PERSON_ID);
+            }
+            catch (ApiException e)
+            {
+                caughtException = e;
+            }
+            
+            Assert.NotNull(caughtException);
```

Comment:
You will not need this first assert @OGKevin , also it is a good practice to keep unit tests with a single assertion. 👍

Reply:
If we remove this first assertion and `caughtException` turns out to be null, the code will break on the next line due to `NullReferenceException`.  Rather have the test fails correctly instead of due to an unhandled exception. 

The good practice of one assertion per test is not always practical.

Reply:
You have a valid point @OGKevin , plus one can say you are being more 'defensive' this way.